domains/hiren.is-cool.dev.json

```diff
@@ -10,7 +10,7 @@
     },
 
     "record": {
-        "CNAME": "hiren2001.github.io"
+        "CNAME": "hiren2001.github.io/me"
```

Comment:
That isn't a valid CNAME.

Reply:
@DEV-DIBSTER but my website https://hiren2001.github.io/me is not showing in hiren.is-cool.dev please can you tell me which settings i need to do in my GitHub repo..

Reply:
Add the domain in your GitHub Pages settings.